From 1fca8243062ef0e19f30f5058eac68b46af7cd5a Mon Sep 17 00:00:00 2001
From: Radha Mohan Chintakuntla <radhac@marvell.com>
Date: Thu, 20 Sep 2018 18:31:54 -0700
Subject: [PATCH 0204/1051] gpio: thunderx: Change the SMC function call
 identifiers

The SMC function identifiers in ATF have now been corrected to use the
SiP range of ids as per the ARM DEN0028B SMC Calling Convention
document. So updating in this driver accordingly.

Signed-off-by: Radha Mohan Chintakuntla <radhac@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/gpio/gpio-thunderx.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpio/gpio-thunderx.c b/drivers/gpio/gpio-thunderx.c
index c6f81abc88d9..f6393de6f89e 100644
--- a/drivers/gpio/gpio-thunderx.c
+++ b/drivers/gpio/gpio-thunderx.c
@@ -68,9 +68,9 @@ static struct task_struct *gpio_installed_tasks[MAX_GPIO];
 
 /* THUNDERX SMC definitons */
 /* X1 - gpio_num, X2 - sp, X3 - cpu, X4 - ttbr0 */
-#define THUNDERX_INSTALL_GPIO_INT       0x43000801
+#define THUNDERX_INSTALL_GPIO_INT       0xc2000801
 /* X1 - gpio_num */
-#define THUNDERX_REMOVE_GPIO_INT        0x43000802
+#define THUNDERX_REMOVE_GPIO_INT        0xc2000802
 
 struct intr_hand {
 	u64	mask;
-- 
2.17.1

